### PR TITLE
Remove dump module

### PR DESCRIPTION
**Pull request by dpino**
The dump module code is stale. It prints out the binding table in the old format. Probably this code will break if gets run (through a signal).

### CODE REVIEW

File: src/apps/lwaftr/lwaftr.lua [hunk 1]
```diff
@@ -581,8 +580,7 @@ function LwAftr:push ()
             -- fixes it, but it's quite a big hammer!
             require('jit').flush()
          elseif msg.kind == messages.lwaftr_message_dump_config then
-            dump.dump_configuration(self)
-            dump.dump_binding_table(self)
+            -- TODO: Dump configuration and binding table to a file.
```
Comment (wingo) [inline src/apps/lwaftr/lwaftr.lua]: Need to remove the dump option from control.lua also, and messages.lua.

Comment (wingo) [inline src/apps/lwaftr/lwaftr.lua]: Or we keep it like it is and update in the future

**Comment by dpino** [comment]
Leave this for now, following kbara's advice on #160.

**Comment by kbara** [comment]
Yes - and when we re-implement it, we need a test that fails if it breaks. It's not worth breaking a code freeze for, but it's worth having consistently working in the future.

**Comment by dpino** [comment]
Closing this PR as #297 invalidates it.

**Pull request closed without merging**